clarity_epp/export/labels.py

```diff
@@ -32,11 +41,40 @@ def storage_location(lims, process_id, output_file):
     process = Process(lims, id=process_id)
 
     # Write header
-    output_file.write('Bakje\tpos\n')
+    output_file.write('Bakje\tpos\r\n')
```

Comment:
I would prefer specification of the encode type. And/or use a function that can write a table based on list/vector, aka `csv.writer` + `writerows`.